Documentation November quick wins (#261)
opendatasoft/ods-documentation: source/publishing_data/03_scheduling_updates/scheduling_updates.rst

Once a dataset is saved with a remote resource, the scheduling tab is activated. The minimum interval is the minute, but it is not activated by default. Please contact Opendatasoft's support if you need minute level scheduling on your domain. You can add as many schedules as you want. For instance, if it fits your needs, you could decide to schedule a dataset to be reprocessed every Monday morning and every Wednesday afternoon.

.. admonition:: Important
   :class: important

>    Schedules are defined to run on the timezone of Paris, France : Central European Time (CET). It means they usually run on GMT+1 and GMT+2 during summer months.

Reviewer: I suggest a small change because GMT+2 is not CET but CEST.
```suggestion
   Schedules are defined to run on the timezone of Paris, France. In standard time, schedules run on GMT+1 (Central European Time). In the summer months, schedules run on GMT+2 (Central European Summer Time).
```
